samples/meeting-av-filters/js/AppPackage/manifest.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.7/MicrosoftTeams.schema.json",
+  "manifestVersion": "devPreview",
+  "version": "1.0.0",
+  "id": "<GUID>",
+  "packageName": "com.microsoft.teams.avfilter",
+  "developer": {
+    "name": "Microsoft",
+    "id": "<GUID>",
+    "privacyUrl": "https://<<BASE-URL>>/privacy",
```

Comment:
The pattern of `privacyUrl` and `termsOfUseUrl` is up to 3rd party developer to decide, and it may differ from `https://<Base-URL>/privacy`.

Reply:
changed it